pkg/mediationcontainer/client_websocket_transport_test.go

```diff
@@ -54,6 +54,10 @@ func TestCreateClientWebSocketTransport(t *testing.T) {
 				LocalAddress: item.localAddress,
 			},
 			item.bindingChannel,
+			SdkProtocolConfig{
+				RegistrationTimeoutSec:        30,
+				ExitProbePodOnProtocolTimeout: false,
```

Comment:
Would `RestartOnRegistrationTimeout` be more clear from an end user's perspective? This is also consistent with your `RegistrationTimeouSec` configuration parameter.